tests/foreman/api/test_host.py

```diff
@@ -1089,6 +1095,465 @@ def test_positive_bootc_api_actions(target_sat, bootc_host, function_ak_with_cv,
     ), "host not included in usage report"
 
 
+@pytest.mark.parametrize(
+    'function_repos_collection_with_manifest',
+    [
+        {
+            'distro': 'rhel10',
+            'YumRepository': [
+                {'url': settings.repos.yum_3.url},
+                {'url': settings.repos.yum_1.url},
+                {'url': settings.repos.yum_6.url},
+            ],
+        }
+    ],
+    indirect=True,
+)
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match('10')
+def test_mixed_usr_overlay_transient_templates(
+    target_sat, bootc_host, function_repos_collection_with_manifest
+):
+    """Verify that you can run an Ansible REX playbook after previously installing a package
+    with the --transient flag.
+
+    :id: e400d595-c503-492c-a922-d9b652558605
+
+    :steps:
+        1.Create and register a bootc host.
+        2.Install a package using --transient flag
+        3.Install a package via Ansible REX.
```

Comment:
Formatting of `:steps:` in 5 tests in this changeset should be updated

```suggestion
    :steps:
        1. Create and register a bootc host.
        2. Install a package using --transient flag
        3. Install a package via Ansible REX.
```